### ISSUE TITLE
Typescipt error with version 2.0.0-beta.2

### ISSUE THREAD

**Issue by mlankamp** [Microsoft/ApplicationInsights-JS#807]
When using version 2.0.0-beta.2 and I import using the following statement (in a *.ts file):
`import { ApplicationInsights } from '@microsoft/applicationinsights-web'`

When webpack runs typescript the following error is shown:
```javascript
6:35 Cannot find module '../../shared/AppInsightsCore/types/JavaScriptSDK.Interfaces/ITelemetryItem'.  
4 | import { TelemetryContext } from "@microsoft/applicationinsights-properties-js";  
5 | import { IDependenciesPlugin } from '@microsoft/applicationinsights-dependencies-js';  
6 | import { ICustomProperties } from "../../shared/AppInsightsCore/types/JavaScriptSDK.Interfaces/ITelemetryItem";  
  |                                   ^
`

the shared directory is not part of the npm package

**Comment by jpiyali** [comment]
@mlankamp  -> sorry about the bug, fixing in the next hour.

**Comment by markwolff** [comment]
Thanking for pointing this out. Fix is in `2.0.0-beta.3`.
